src/index.tsx

```diff
@@ -1,26 +1,38 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import Viewer from "./components/Viewer/Viewer";
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import { Vault } from '@hyperion-framework/vault';
+import { Manifest, ManifestNormalized } from '@hyperion-framework/types';
+import Viewer from './components/Viewer/Viewer';
 
 interface Props {
-  manifestId: string;
+  id: string;
 }
+const sampleManifest: string =
+  'https://raw.githubusercontent.com/mathewjordan/mirador-playground/main/assets/iiif/manifest/new_airliner_invalid.json';
 
-const App: React.FC<Props> = ({ manifestId }) => {
-  return (
-    <Viewer
-      manifest={{
-        label: {
-          en: ["manifest.label"],
-        },
-      }}
-    />
-  );
+const App: React.FC<Props> = ({ id }) => {
+  const [manifest, setManifest] = useState<ManifestNormalized | undefined>(undefined);
+  const [loaded, setLoaded] = useState(false);
+  useEffect(() => {
+    const vault = new Vault();
```

Comment:
Is it overkill here to put a check in for a truthy `loaded` value?

Reply:
@adamjarling I'm all for it, can we chat first?